apps/protailwind/src/video/exercise-overlay.tsx

```diff
@@ -438,16 +437,6 @@ const BlockedOverlay = () => {
         </>
       ) : (
         <>
-          {videoResourceId && (
-            <Image
-              src={thumbnail}
-              layout="fill"
-              alt=""
-              aria-hidden="true"
-              objectFit="cover"
-              className="opacity-50 blur-sm brightness-50 contrast-125"
-            />
-          )}
```

Comment:
why is this being removed?

Reply:
it's being used within the blocked overlay
<img width="500" alt="image" src="https://user-images.githubusercontent.com/25487857/214338315-f24b9915-e481-4f92-919c-ab5ff382ac57.png">

Reply:
I will add it back! didn't realize that part